packages/perseus/src/widgets/mock-widgets/mock-widget.tsx

```diff
@@ -6,18 +6,15 @@ import * as React from "react";
 import {getPromptJSON as _getPromptJSON} from "../../widget-ai-utils/mock-widget/prompt-utils";
 
 import scoreMockWidget from "./score-mock-widget";
+import validateMockWidget from "./validate-mock-widget";
```

Comment:
Not related to your PR, but I wonder if all this mock widget stuff should even be in `/packages` when it could possibly live in `/testing`.